library/billing/src/Claim.php

```diff
@@ -53,6 +39,26 @@ class Claim
     var $copay;             // total of copays from the ar_activity table
     var $facilityService;
 
+    // This enforces the X12 Basic Character Set. Page A2.
+    private function x12clean($str)
```

Comment:
btw, this is a good exercise to learn how to do search/replace within a script in phpstorm.